src/scene/container/bounds/Bounds.ts

```diff
@@ -253,7 +253,7 @@ export class Bounds
      * Resizes the bounds object to include the given rectangle.
      * @param rect - The rectangle to be included.
      */
-    public fit(rect: Rectangle): this
+    public fitRect(rect: Rectangle): this
```

Comment:
This seems like a needless breaking API change

Can we not rename the other method instead

Reply:
agree

Reply:
good shout!